cmd/create-account/main.go

```diff
@@ -83,7 +81,14 @@ func main() {
 		os.Exit(1)
 	}
 
-	pass := getPassword(password, pwdFile, pwdStdin, askPass, os.Stdin)
+	if len(fmt.Sprintf("@%s:%s", *username, cfg.Global.ServerName)) > 255 {
+		logrus.Fatalln("Username can not be longer than 255 characters.")
```

Comment:
Needs better message here as they may not realise the server name is part of the 255. Log the entire username in the error please.